e2e/suite_test.go

```diff
@@ -134,13 +134,14 @@ var _ = Describe("TopoLVM", func() {
 			}).Should(Succeed())
 		})
 
-		sanity.GinkgoTest(&sanity.Config{
-			Address:           "/tmp/topolvm/worker1/plugins/topolvm.cybozu.com/node/csi-topolvm.sock",
-			ControllerAddress: "/tmp/topolvm/worker1/plugins/topolvm.cybozu.com/controller/csi-topolvm.sock",
-			TargetPath:        "/tmp/topolvm/worker1/plugins/topolvm.cybozu.com/node/mountdir",
-			StagingPath:       "/tmp/topolvm/worker1/plugins/topolvm.cybozu.com/node/stagingdir",
-			TestVolumeSize:    1073741824,
-			IDGen:             &sanity.DefaultIDGenerator{},
-		})
+		tc := sanity.NewTestConfig()
+		tc.Address = "/tmp/topolvm/worker1/plugins/topolvm.cybozu.com/node/csi-topolvm.sock"
+		tc.ControllerAddress = "/tmp/topolvm/worker1/plugins/topolvm.cybozu.com/controller/csi-topolvm.sock"
+		tc.TargetPath = "/tmp/topolvm/worker1/plugins/topolvm.cybozu.com/node/mountdir"
+		tc.StagingPath = "/tmp/topolvm/worker1/plugins/topolvm.cybozu.com/node/stagingdir"
+		tc.TestVolumeSize = 1073741824
+		tc.IDGen = &sanity.DefaultIDGenerator{}
+		tc.IdempotentCount = 0
```

Comment:
This is to skip idempotency tests.
I think we should make them passed.
https://pkg.go.dev/github.com/kubernetes-csi/csi-test/v4/pkg/sanity#TestConfig

Reply:
Agreed.
I inspected a failed test and found an error message that is `LogicalVolume.topolvm.cybozu.com "sanity-node-full-f497324e-e3ef8e64" not found`.
https://github.com/topolvm/topolvm/runs/1802349235
It seems that CreateVolume is failed or getting created LogicalVolume is failed.

Reply:
I'm not sure that `LogicalVolume.topolvm.cybozu.com` is correct.
In CRD, I can not find it, but found `plural: logicalvolumes`.